extensions/ftp/src/main/java/org/mule/extension/ftp/internal/sftp/connection/SftpConnectionProvider.java

```diff
@@ -61,6 +62,14 @@
   @Optional
   private String knownHostsFile;
 
+  /**
+   * If provided, a proxy will be used for the connection.
+   */
+  @Parameter
+  @Optional
+  @Alias("sftp-proxy-config")
```

Comment:
disregard this comment. Better to have the redundant prefix in case we ever want to do this for classic ftp as well